### PR TITLE
[23.0] Display dialog when saving workflow with invalid connections

### PR DESCRIPTION
**Pull request by mvdbeek**
Closes https://github.com/galaxyproject/galaxy/issues/15753

Screenshot 2023-03-17 at 15 41 02

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by wm75** [comment]
Maybe "seems to have" since probably only the wf editor thinks so?

**Comment by mvdbeek** [comment]
That we'll fix, hopefully ;)

**Comment by wm75** [comment]
and maybe "Save" or "Save Workflow" to reflect the button's purpose?

File: client/src/components/Workflow/Editor/Options.vue [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+import { BDropdown, BDropdownItem, BButton } from "bootstrap-vue";
+import { useConfirmDialog } from "@/composables/confirmDialog";
+
+const emit = defineEmits<{
+    (e: "onAttributes"): void;
+    (e: "onSave"): void;
+    (e: "onReport"): void;
+    (e: "onSaveAs"): void;
+    (e: "onLayout"): void;
+    (e: "onUpgrade"): void;
```
Comment (dannon) [inline client/src/components/Workflow/Editor/Options.vue:13]:
```suggestion
    (e: "onLint"): void;
    (e: "onUpgrade"): void;
```

**Review by dannon — APPROVED** [review]

**Pull request merged**